engine/driver/src/test/java/org/opennms/oce/engine/driver/ScoringStrategyTest.java

```diff
@@ -77,7 +77,7 @@ public void canComputeScores() {
         report = scorer.score(Sets.newHashSet(emtpyIncident), Sets.newHashSet(emtpyIncident));
         assertThat(report.getScore(), closeTo(0.0d, delta));
 
-        // Comparing an incident with a single alarm to an empty incident should generate a score greather than 0
+        // Comparing an incident with a single alarm to an empty incident should generate a score greater than 0
```

Comment:
This does not align with what the Set Intersection strategy is doing and is the only assertion that prevents the SetIntersectionScoringStrategy from being tested by this testCase